src/main/resources/com/sonyericsson/rebuild/RebuildAction/parameterized.jelly

```diff
@@ -27,7 +27,23 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
-         xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:local="local">
+    <d:taglib uri="local">
+        <d:tag name="blockWrapperTBody">
+            <j:choose>
+                <j:when test="${divBasedFormLayout}">
+                    <div>
```

Comment:
normally you can just remove this completely tbody is quite uncommon in forms for Jenkins if it works it's fine though

Reply:
Hi @timja,

Thank you for your review.

So, is it better to remove the `tbody` tag directly instead of creating and using the `blockWrapperTBody`?

Reply:
I think so (but test it)

Reply:
Done :+1: